platform/ui/src/components/MeasurementTable/MeasurementTable.jsx

```diff
@@ -3,6 +3,70 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { Icon } from '@ohif/ui';
 
+const MeasurementItem = ({ id, index, label, displayText, isActive, onClick, onEdit }) => {
```

Comment:
Can we use `function` and kick this to a separate file? Doesn't need to be exported by `@ohif/ui`, just consumed by this component.

Reply:
Sure! updating.